lib/docker.js

```diff
@@ -149,13 +161,13 @@ function createDockerFile() {
     }
 
     if (opts.deploy) {
-        contents += 'CMD /usr/bin/npm install --production && /usr/bin/npm install heapdump';
+        contents += 'CMD npm install --production && npm install heapdump';
```

Comment:
Actually, you need to check the value of `nodeVersion` before setting this and set it iff `nodeVersion && nodeVersion !== 'system'`